plugins/cindygl/src/js/CanvasWrapper.js

```diff
@@ -4,22 +4,21 @@
 function addCanvasWrapperIfRequired(name, api) {
   if (!canvaswrappers.hasOwnProperty(name)) {
     let img = api.getImage(name, true); //this might be a canvas as well
```

Comment:
CindyGL, like Cindy3D, will be cross-compiled by Closure from ES6 to ES5. It has ES6 syntax all over the place.